src/components/RekognitionConnectionLocal.ts

```diff
@@ -11,8 +16,17 @@ export default class RekognitionConnectionLocal extends BaseRekognitionConnectio
 
   getClient (): Rekognition {
     return {
-      detectFaces: () => {
-        return;
+      detectFaces: (params) => {
+        const allMocks = {
+          'image.jpg': mockResponse1,
+          'image2.jpg': mockResponse2,
+          'image3.jpg': mockResponse3,
+          'image4.jpg': mockResponse4,
+        };
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
```

Comment:
I think it will work at this point without these two lines if you fixed the params :)